espnet2/train/preprocessor.py

```diff
@@ -169,6 +170,20 @@ def __init__(
                 token_list=token_list,
                 unk_symbol=unk_symbol,
             )
+            if transcript_token_list is not None:
```

Comment:
It seems to be task-specific. How about we just make it a new class?

Reply:
Sure, I have made a new preprocessor class using the CommonPreprocessor class as the base class.